sdks/python/apache_beam/io/filesystem.py

```diff
@@ -330,6 +334,10 @@ def _get_compression_type(path, compression_type):
                       'was %s' % type(compression_type))
     return compression_type
 
+  @staticmethod
```

Comment:
Should this be in FileSystem (CompressedFile is an abstraction added by SDK on top of the files returned by a FileSystem) ?